src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -858,6 +857,43 @@ public void canSearchForItemsByTags()
     assertThat(itemTags.get("tagList"), hasItem(TAG_VALUE));
   }
 
+  @Test
+  public void canSearchForItemsByStatus()
+    throws MalformedURLException,
+    InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    UnsupportedEncodingException {
+
+    UUID holdingsRecordId = createInstanceAndHolding(mainLibraryLocationId);
+
+    createItem(smallAngryPlanet(UUID.randomUUID(), holdingsRecordId));
+    createItem(nod(UUID.randomUUID(), holdingsRecordId));
+    createItem(uprooted(UUID.randomUUID(), holdingsRecordId));
+    createItem(temeraire(UUID.randomUUID(), holdingsRecordId));
+    createItem(interestingTimes(UUID.randomUUID(), holdingsRecordId));
+
+    CompletableFuture<Response> searchCompleted = new CompletableFuture<>();
+
+    String url = itemsStorageUrl("") + "?query=" + URLEncoder.encode("status.name==\"Available\"",
+      StandardCharsets.UTF_8.name());
```

Comment:
Can you replace

    URLEncoder.encode(s, StandardCharsets.UTF_8.name())

by

    urlEncode(s)

?